ci/aws-rds-storage.yml

```diff
@@ -1,9 +1,11 @@
 ---
 platform: linux
 image_resource:
-  type: docker-image
+  type: registry-image
   source:
     repository: kennethreitz/pipenv
```

Comment:
Can we push this to the mirror from the jumpbox?

Reply:
not sure what you mean - are you asking if this image can be mirrored through our registry mirror?

Reply:
Do we have to push `kennethreitz/pipenv` first to the mirrored registry to be able to pull it right away?

Reply:
no - the mirror is a pull-through cache mirror, so if we try to pull an image and it doesn't exist, the mirror pull it from docker hub and store it locally, then serve the local copy to us.

Reply:
Copy that.  👍